addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -314,15 +314,20 @@ def _l10n_es_tbai_get_invoice_values(self, invoice, cancel):
 
         # Lines (detalle)
         refund_sign = (1 if values['is_refund'] else -1)
-        values['invoice_lines'] = [{
-            'line': line,
-            'discount': (discount := (line.balance / (1 - line.discount / 100) * line.discount / 100)) * refund_sign,
-            'unit_price': (line.balance + discount) / line.quantity * refund_sign,
-            'total': line.price_total * abs(line.balance / line.amount_currency if line.amount_currency != 0 else 1) * -refund_sign if not any(
-                [t.l10n_es_type == 'sujeto_isp' for t in line.tax_ids]) else abs(line.balance) * -refund_sign * (-1 if line.price_total < 0 else 1),
-            'description': regex_sub(r'[^0-9a-zA-Z ]', '', line.name)[:250],  # only keep characters allowed in description
```

Comment:
Maybe it's not the most useful comment, but i'd rather improve it than remove it, e.g.
> description only allows up to 250 alphanumeric characters.

Reply:
I mean I wrote the suggestion and everything but it's not like a necessary comment. Whatever.